src/wechaty.ts

```diff
@@ -282,6 +327,8 @@ export class Wechaty extends EventEmitter implements Sayable {
     // No need to set puppet/wechaty, so do not clone
     this.UrlLink        = UrlLink
     this.MiniProgram    = MiniProgram
+
+    this.installGloablPlugin()
```

Comment:
This will make all the Wechaty instances be affected by the other instances, it's more like to be introducing problems than the benefits here.

Reply:
I don't get it.

Reply:
Never mind, I misunderstood your code.

`this.installGlobalPlugin()` is good.